.travis.yml

```diff
@@ -1,3 +1,14 @@
 language: erlang
 otp_release:
-   - 17.3
+  - 18.1
```

Comment:
Interestingly Travis still built even when I specified 18.3 in one of our apps:

```
18.3 is not installed. Downloading and installing pre-build binary.
wget -o $HOME/erlang.tar.bz2 ${archive_url}
mkdir -p ~/otp && tar -xf erlang-18.3-nonroot.tar.bz2 -C ~/otp/
```

I wonder what it *actually* built with?